content/iter8 trend/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2020-08-11T12:00:00+00:00
+menuTitle: iter8-trend
+title: Getting Started with iter8-trend
+weight: 25
+summary: Describes how to install and use iter8-trend
```

Comment:
Just for note for the future, 

If we don't plan on auto generating our list files, then we don't need to fill out the `summary` front matter field anymore. Feel free to do so anyway if you'd like.

Reply:
@kalantar @sriumcp